res/supersonic-desktop.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Version=1.0
```

Comment:
Should this be the same as the app version? (i.e. currently 0.1.0-beta, but next version will probably be 0.1.0  - dropping the beta)

Reply:
Nevermind, just found out this is the version of the .desktop standard